### PR TITLE
when clicked, conversation appears with messages

### PR DESCRIPTION
**Pull request by ianhunterharold**

### CODE REVIEW

**Review by renburnett — APPROVED** [review]
looks great!

File: src/containers/ChatWindow.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
 import React, { PureComponent } from 'react'
 import MessageForm from './MessageForm'
+import Message from '../components/Message'
 import PrivacyHOC from '../HOC/PrivacyHOC'
 
 class ChatWindow extends PureComponent {
+
+  displayConversationMessages = () => {
+    return this.props.currentConversation.messages.map((message) => {
+      return <Message message={message}/>
```
Comment (kel29) [inline src/containers/ChatWindow.js:10]: message?

**Review by kel29 — APPROVED** [review]
all good

File: src/containers/ChatWindow.js (hunk 1 above)
Comment (renburnett) [reply src/containers/ChatWindow.js:10]: yes?

**Pull request merged**